cosmo_tester/framework/test_hosts.py

```diff
@@ -165,57 +169,73 @@ def upload_necessary_files(self):
     def ssh_key(self):
         return self._ssh_key
 
-    def get_remote_file(self, remote_path, local_path, use_sudo=True):
+    def get_remote_file(self, remote_path, local_path):
         """ Dump the contents of the remote file into the local path """
+        # Similar to the way fabric1 did it
+        remote_tmp = '/tmp/' + hashlib.sha1(remote_path).hexdigest()
+        self.run_command(
+            'cp {} {}'.format(remote_path, remote_tmp),
+            use_sudo=True,
+        ),
 
         with self.ssh() as fabric_ssh:
             fabric_ssh.get(
-                remote_path,
+                remote_tmp,
                 local_path,
-                use_sudo=use_sudo
```

Comment:
@geokala just noticed that `use_sudo` is removed, is this because of the new fabric upgrade ?

Reply:
It's removed because it was defaulted to True and never set to False anywhere, so seemed a bit pointless.